trade-finance-manager-api/src/constants/currency.js

```diff
@@ -0,0 +1,7 @@
+const CURRENCY = {
```

Comment:
the file name should be `currency.constant.js`

Reply:
done